### PR TITLE
Documentation enhancements

### PR DESCRIPTION
**Pull request by grigorovici-andreea**
Added bindings information in build instructions in Windows and Linux.

Signed-off-by: Andreea Grigorovici <andreea.grigorovici@analog.com>

### CODE REVIEW

File: doc/linux/build_instructions.md [hunk 1]
```diff
@@ -73,3 +73,8 @@ mkdir build && cd build
 cmake -DWITH_EXAMPLES=on -DCMAKE_PREFIX_PATH="/opt/glog;/opt/protobuf;/opt/websockets" ..
 make
 ```
+
+## SDK with bindings
+
+- Please check the readme files for each type of binding in the <a href="https://github.com/analogdevicesinc/aditof_sdk/tree/master/bindings"> bindings directory </a>.
```
Comment (dNechita) [inline doc/linux/build_instructions.md]: Is the hyperlink really necessary?

Comment (grigorovici-andreea) [reply doc/linux/build_instructions.md]: I thought it is easier for the user this way. I can take it out if you think it is more confusing than helpful.

Comment (dNechita) [reply doc/linux/build_instructions.md]: But what is easier?

Comment (grigorovici-andreea) [reply doc/linux/build_instructions.md]: to get to the right directory

Comment (dNechita) [reply doc/linux/build_instructions.md]: I meant the using of 'href'. You can easily do this without html.

Comment (grigorovici-andreea) [reply doc/linux/build_instructions.md]: ok, solved.

**Pull request merged**